backend/Dockerfile

```diff
@@ -14,7 +14,7 @@ COPY Pipfile Pipfile.lock ./
 
 RUN pip install --upgrade pip \
     && pip install pipenv==v2020.11.15 \
-    && pipenv install --system --deploy
+    && pipenv install --dev --system --deploy --verbose
```

Comment:
I added verbose flag to better understand what's going on, can be removed because the output turned out to not be very useful, but it might come in handy at some point.